sentry-actix/src/lib.rs

```diff
@@ -211,6 +261,9 @@ impl<S: 'static> Middleware<S> for SentryMiddleware {
         let cached_data = Arc::new(Mutex::new(None));
 
         let root_scope = hub.push_scope();
+        if self.track_session {
+            hub.start_session();
+        }
```

Comment:
How does it stop?

Reply:
It stops when the scope goes out of scope :-D